executor-service/src/main/java/executor/service/App.java

```diff
@@ -1,13 +1,16 @@
 package executor.service;
 
 import executor.service.facade.parallel.ParallelFlowExecutorService;
-import executor.service.factory.di.ApplicationContext;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@SpringBootApplication
 public class App {
```

Comment:
You can implement a `CommandLineRunner` and place the logic inside the run method, 
which will be executed when the Spring Boot application starts.

```
    @Override
    public void run(String... args) {
        executorService.execute();
    }
```

Reply:
done it, thanks!